README.md

```diff
@@ -0,0 +1,68 @@
+<p align='center'>
+    <h2  align='center'><span>Sgp4 for </span><span  style="color: #654ef1">WAPM</span></h2>
+    <p align='center'>
+    <a title="Hosted on WAPM" href="https://wapm.io/dynamite-bud/sgp4">
+    <img src="https://img.shields.io/badge/WAPM-654ef1.svg?style=flat-square&logo=WebAssembly&labelColor=654ef1&logoColor=fff">
+    </a>
+    <a title="GitHub license button" href="https://github.com/wasmerio/sgp4/blob/main/LICENSE_MIT.md"><img src="https://img.shields.io/github/license/reklatsmasters/vscode-wasm.svg?style=flat-square"></a>
+    <a title="GitHub Apache license button" href="https://github.com/wasmerio/sgp4/blob/main/LICENSE_APACHE.md"><img src="https://img.shields.io/badge/License-Apache-blue.svg?style=flat-square"></a>
+  </p>
+</p>
+
+---
+
+This repository is the port of the [SGP4](https://crates.io/crates/sgp4) library to WebAssembly.
+
+Published package is available on [WAPM](https://wapm.io/dynamite-bud/sgp4).
+
+Package is published on the following platforms:
```

Comment:
I think this list is a bit misleading. It's only actually published to WAPM. It's just that WAPM provides Python and JavaScript bindings. We might want to remove it for now.